name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/traderepublic/TradeRepublicPDFExtractorTest.java

```diff
@@ -1351,16 +1351,22 @@ public void testKontoauszug05()
                         hasSource("Kontoauszug05.txt"), hasNote(null))));
 
         // assert transaction
-        assertThat(results, hasItem(interest(hasDate("2023-04-01"), hasAmount("EUR", 0.01), //
-                        hasSource("Kontoauszug05.txt"), hasNote(null))));
+        assertThat(results, hasItem(withFailureMessage( //
+                        Messages.MsgErrorTransactionTypeNotSupported,  //
```

Comment:
Hello @christen90 
I understand the reason, but we should issue a different message here, which could possibly also be used for other PDF importers.

**I could imagine the following message:**
```
MsgErrorTransactionAlternativeDocumentRequired = "Die Buchung wird mit diesem Dokument nicht verarbeitet. Ihre Bank oder Ihr Broker stellt ein anderes Dokument dafür zur Verfügung."

MsgErrorTransactionAlternativeDocumentRequired = "This document does not process the booking. Your bank or broker will provide a different document for this purpose."
```

Please change the error message and add the missing translations via deepl.com.

Regards
Alex 😃